end-to-end-tests/specs/home.spec.js

```diff
@@ -259,4 +259,36 @@ describe('single study query', function() {
 
         });
     });
+
+    describe('enrichments', function() {
+        it('should show mutations plot', function() {
+            browser.url(`${CBIOPORTAL_URL}/index.do?cancer_study_id=ov_tcga_pub&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=ov_tcga_pub_cna_seq&gene_list=BRCA1+BRCA2&geneset_list=+&tab_index=tab_visualize&Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=ov_tcga_pub_gistic`);
+
+            // click enrichments tab
+            $('#enrichments-result-tab').waitForExist(30000);
+            $('#enrichments-result-tab').click();
+
+            // wait for plot to show
+            $('#ov_tcga_pub_mutations_plot_div').waitForExist(60000);
+            assert(true);
```

Comment:
i wonder if we even needs this assert?  would be good to know.

Reply:
@alisman yes we can remove it